geodirectory-functions/map-functions/map_functions.php

```diff
@@ -69,7 +126,20 @@ function send_marker_jason_to_js()
     }
 }
 
-/* Home map Taxonomy walker */
+/**
+ * Home map Taxonomy walker
+ *
+ * @since 1.0.0
+ * @package GeoDirectory
+ * @param string $cat_taxonomy Name of the taxonomy e.g place_category
+ * @param int $cat_parent Parent term ID to retrieve its child terms. Default 0

```

Comment:
$cat_parent is optional so should be showen to be.
@param int $cat_parent Optional. Parent term ID to retrieve its child terms. Default 0.